Source/OpenWareMidiControl.h

```diff
@@ -102,6 +103,7 @@ enum OpenWareMidiSysexCommand {
   SYSEX_FLASH_ERASE               = 0x14,
   SYSEX_SETTINGS_RESET            = 0x15,
   SYSEX_SETTINGS_STORE            = 0x16,
+  SYSEX_FIRMWARE_SAVE             = 0x17,

```

Comment:
Can we have SYSEX_FIRMWARE_LOAD and support for downloading resources from device? This was brought up in the forum already as a desired feature.

Reply:
As a future addition, yes, when there's a sensible and safe way for patches to store resources too.

Reply:
Not sure why we would need this. I was thinking about just being able to download individual resource or patch back from the device. Either that or a single command for downloading all storage contents in a single sysex blob. Either way, I think that some form of backup is necessary. This would also make it possible to migrate patches to another device.

Reply:
Okay yes, for backup it would also make sense. I was thinking about downloading data stored by the patch, such as a sequence or sample.